src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -648,4 +663,12 @@ public ECKey getMyKey() {
 
     return ECKey.fromPrivate(Hex.decode(INSTANCE.p2pNodeId));
   }
+
+  private static void initBackupProperty(Config config) {
+    INSTANCE.backupPriority = config.hasPath("node.backup.priority")
+        ? config.getInt("node.backup.priority") : new Random(1000).nextInt();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a cryptographically strong random number generator (RNG) like "java.security.SecureRandom" in place of this PRNG ([squid:S2245](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2245))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=951&issues=AWPTJmyCA0k93xG_IQR0&open=AWPTJmyCA0k93xG_IQR0)

[ISSUE_KEY]: AWPTJmyCA0k93xG_IQR0